src/components/KeyboardShortcutsModal.js

```diff
@@ -138,7 +139,7 @@ function KeyboardShortcutsModal({isShortcutsModalOpen = false, isSmallScreenWidt
         };
         // We only want this to run on mount
         // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, []);
+    }, [KeyboardShortcut.getDocumentedShortcuts]);
```

Comment:
Are we still depending on `KeyboardShortcut.getDocumentedShortcuts`? The comment says that this `useEffect` only runs on mount.